minefana-bungeecord/src/main/java/com/github/games647/minefana/collectors/BungeePlayerCollector.java

```diff
@@ -50,7 +54,15 @@ public void run() {
 
     @Override
     protected String getLocale(ProxiedPlayer player) {
-        return player.getDisplayName();
+        String locale;
+        try{
+            locale = player.getLocale().getDisplayName();
```

Comment:
Same as below

Reply:
Here the player.getLocale() is always null not sure maybe I am missing something

Reply:
error looks like this https://pastebin.com/UNvRwkpE

Reply:
Found the source. The locale and server isn't initialized at that time. We are currently fetching the data directly on join. We should delay it to ~3 seconds. Then should both data be ready.